cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -556,3 +557,57 @@ func restoreEnv(e map[string]string) {
 		os.Setenv(k, v)
 	}
 }
+
+func TestKubeletVersionCheck(t *testing.T) {
+	type T struct {
+		kubeletVersion string
+		k8sVersion     string
+		expectErrors   bool
+		expectWarnings bool
+	}
+
+	cases := []T{
+		{"v1.7.3", "v1.7.8", true, false},          // Too old kubelet
```

Comment:
this confused me first: please say `// too old kubelet generally, v1.7 kubelets not supported anymore`

Reply:
That is intentional, to check error path with `kubeadmconstants.MinimumKubeletVersion`

Reply:
yeah, but I'd appreciate if you changed the comment to be clearer. That was my intention